actuator-firmware/src/servo.c

```diff
@@ -100,6 +95,9 @@ THD_FUNCTION(servo_thd, arg)
 
 void servo_start(void)
 {
-    static THD_WORKING_AREA(servo_thd_wa, 256);
+    static THD_WORKING_AREA(servo_thd_wa, 512);
+
+    servo_init();
```

Comment:
`servo_init()` -> `timer_init()`

Reply:
servo_init does more than timer_init, and is a separate function.